analyzers/its/SingleRule.ruleset.template

```diff
@@ -1152,7 +1152,6 @@
     <Rule Id="S1142" Action="None" />
     <Rule Id="S1143" Action="None" />
     <Rule Id="S1144" Action="None" />
-    <Rule Id="S1145" Action="None" />
```

Comment:
same comment as for `analyzers/its/AllSonarAnalyzerRules.ruleset` - this file is generated without being too smart about which rules are actually implemented

this is only used to activate a single rule during ITs - if someone activates `S9998` it won't do anything

I believe you should revert it